### PR TITLE
[build] Android: fix GUI not loading on Qt 5.15

### PR DESCRIPTION
**Pull request by icota**
PR moved from https://github.com/bitcoin-core/gui/pull/504 as it escaped the confines of `src/qt`.

### CODE REVIEW

**Comment by hebasto** [comment]
ACK d09d14c1d31e14b69abcf0d030e87b475bd084a0

Screenshot_1639389537

Screenshot_1639389549

The excerpt from the `/data/user/0/org.bitcoincore.qt/files/.bitcoin/testnet3/debug.log`:

```
2021-12-13T10:02:09Z Bitcoin Core version v22.99.0-d09d14c1d31e (release build)
2021-12-13T10:02:09Z Qt 5.15.2 (static), plugin=android (static)
2021-12-13T10:02:09Z Static plugins:
2021-12-13T10:02:09Z  QAndroidPlatformIntegrationPlugin, version 331520
2021-12-13T10:02:09Z Style: fusion / QFusionStyle
2021-12-13T10:02:09Z System: Android  (11.0), x86_64-little_endian-lp64
2021-12-13T10:02:09Z Screen:  1280x844, pixel ratio=2.0
```

My Guix builds are coming...

File: src/qt/test/test_main.cpp [hunk 1]
```diff
@@ -34,6 +34,8 @@ Q_IMPORT_PLUGIN(QXcbIntegrationPlugin);
 Q_IMPORT_PLUGIN(QWindowsIntegrationPlugin);
 #elif defined(QT_QPA_PLATFORM_COCOA)
 Q_IMPORT_PLUGIN(QCocoaIntegrationPlugin);
+#elif defined(QT_QPA_PLATFORM_ANDROID)
+Q_IMPORT_PLUGIN(QAndroidPlatformIntegrationPlugin);
```
Comment (promag) [inline src/qt/test/test_main.cpp:38]: fec72387c587fd257df89151ca37aa42583aff34

nit could ditch `;`.

File: src/Makefile.qt.include [hunk 2]
```diff
@@ -394,8 +394,7 @@ bitcoin_qt_apk: FORCE
 	cp $(dir $(lastword $(CC)))../sysroot/usr/lib/$(host_alias)/libc++_shared.so $(APK_LIB_DIR)
 	tar xf $(QT_BASE_PATH) -C qt/android/src/ $(QT_BASE_TLD)src/android/jar/src --strip-components=5
 	tar xf $(QT_BASE_PATH) -C qt/android/src/ $(QT_BASE_TLD)src/android/java/src --strip-components=5
-	tar xf $(QT_BASE_PATH) -C qt/android/res/ $(QT_BASE_TLD)src/android/java/res --strip-components=5
```
Comment (promag) [inline src/Makefile.qt.include:397]: d09d14c1d31e14b69abcf0d030e87b475bd084a0

Why is this line removed?

File: src/qt/android/AndroidManifest.xml [hunk 3]
```diff
@@ -32,7 +32,9 @@
             <meta-data android:name="android.app.background_running" android:value="true"/>
             <meta-data android:name="android.app.auto_screen_scale_factor" android:value="true"/>
             <meta-data android:name="android.app.extract_android_style" android:value="default"/>
-    </activity>
+
```
Comment (promag) [inline src/qt/android/AndroidManifest.xml:35]: d09d14c1d31e14b69abcf0d030e87b475bd084a0

nit, remove empty line?

**Review by promag — COMMENTED** [review]
Concept ACK

File: src/Makefile.qt.include (hunk 2 above)
Comment (icota) [reply src/Makefile.qt.include:397]: Because I found out that the only thing we are extracting from there are the strings (in many languages) to do with Ministro (which is a way of fetching Qt libs at runtime). This wasn't used and isn't likely to be in the furure. That whole directory used to be .gitignored but now we have `values/libs.xml` there so two birds one stone

**Review by hebasto — APPROVED** [review]
re-ACK 27f353d8efca19bc2f7dc79b09d4737c9401a768

GUIX builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
3a1f3ff50da24bf8d5565654947d1c6c5498f60e7470613cd9925f128714c88c  guix-build-27f353d8efca/output/aarch64-linux-gnu/SHA256SUMS.part
d091175a6883e56a8b2b4701807acb29b626fdc1154dc5e4f8dd64e0861f46af  guix-build-27f353d8efca/output/aarch64-linux-gnu/bitcoin-27f353d8efca-aarch64-linux-gnu-debug.tar.gz
bcb5278ef3f22c98c39bf840092fe38149c0f04423922d048659d3ffc763688a  guix-build-27f353d8efca/output/aarch64-linux-gnu/bitcoin-27f353d8efca-aarch64-linux-gnu.tar.gz
6c9ba74a7df5dcfb21b2580b0e93599316c2fa96d569860f9810ea2028553c5b  guix-build-27f353d8efca/output/arm-linux-gnueabihf/SHA256SUMS.part
f8e19337174d3e44b165dfadc52c8fbe95d127aa35c5240a6b7b145a63741900  guix-build-27f353d8efca/output/arm-linux-gnueabihf/bitcoin-27f353d8efca-arm-linux-gnueabihf-debug.tar.gz
c1fa9210e0f928885912f13a3db032d2600427d17a1a0184807e584cb15e76cb  guix-build-27f353d8efca/output/arm-linux-gnueabihf/bitcoin-27f353d8efca-arm-linux-gnueabihf.tar.gz
1d7e694023e68f420b4b59f6ddcab837a98be69a88fdcc7328a1f5105731bad8  guix-build-27f353d8efca/output/dist-archive/bitcoin-27f353d8efca.tar.gz
18ccbfd3d3d5bc8a5d1492f72ee6e28343993337e755a045e94381cbd370893a  guix-build-27f353d8efca/output/powerpc64-linux-gnu/SHA256SUMS.part
e07a198680870e8e37d59c69a624adf2771933674d3baad08f2faaa9efec3740  guix-build-27f353d8efca/output/powerpc64-linux-gnu/bitcoin-27f353d8efca-powerpc64-linux-gnu-debug.tar.gz
32d76d7a5905a9230a7c627785e2ff8ab978df8a54eca82fcdfc07b333bfcb27  guix-build-27f353d8efca/output/powerpc64-linux-gnu/bitcoin-27f353d8efca-powerpc64-linux-gnu.tar.gz
08b8c525a7903c929d1e3b25d4fdce2c36a3072f0f49d9873700352a007933f5  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/SHA256SUMS.part
105c6de1adf373fc17c1cd1b68913e5d0b7fb14b67b56cf91251406c8dca0fb2  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/bitcoin-27f353d8efca-powerpc64le-linux-gnu-debug.tar.gz
7b8809c31d6aeeac078baf3c13f7ad814c1e23e536caaefcf827b1ca4451a25d  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/bitcoin-27f353d8efca-powerpc64le-linux-gnu.tar.gz
7dfb0cdf16770036fdc05824659dfb0129d9f422dc18a8d28d2472141bd5cbcb  guix-build-27f353d8efca/output/riscv64-linux-gnu/SHA256SUMS.part
9cd3e13843f2b7e6a4c8f9e25b5f296fd860e69ae61dd2d394f181d44b0167eb  guix-build-27f353d8efca/output/riscv64-linux-gnu/bitcoin-27f353d8efca-riscv64-linux-gnu-debug.tar.gz
cd91cbd7ffa19f2bdd0be75468941823e964a21f25fb03f225f6ebef482559d5  guix-build-27f353d8efca/output/riscv64-linux-gnu/bitcoin-27f353d8efca-riscv64-linux-gnu.tar.gz
394edcf459ebfc17676675b2d20cb76d14024f70e88c19d0647b512bd05ceb2f  guix-build-27f353d8efca/output/x86_64-apple-darwin/SHA256SUMS.part
049ecd19d3914938b6c5d88c2f334b0c455241f83788972a14d7eaa4fdbd32b8  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx-unsigned.dmg
3b9177de6381c7d2e22171d7a5c6701138015570c07e9ad35f2665b5dc20d9a9  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx-unsigned.tar.gz
a30168c298d934127b5a724907ae1feb24aa9f4807a6769f296efb3f5113ef77  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx64.tar.gz
42d2ded35814b6cac353f304fc6cc68e029956363fa4d4d8a04052f0e3116fd3  guix-build-27f353d8efca/output/x86_64-linux-gnu/SHA256SUMS.part
3eaabaee1b3a81f7dc6be9ab78b1b6bb3cb41b95dcae95597a03bf2e97e378b1  guix-build-27f353d8efca/output/x86_64-linux-gnu/bitcoin-27f353d8efca-x86_64-linux-gnu-debug.tar.gz
b352a51d5af1c9bdf9f48016d7436759fc16b18c389456a855e322b508da90d2  guix-build-27f353d8efca/output/x86_64-linux-gnu/bitcoin-27f353d8efca-x86_64-linux-gnu.tar.gz
673285ec5b54a8e1a6b5e109c458622462b33355872146557e1bc35f57e1cf23  guix-build-27f353d8efca/output/x86_64-w64-mingw32/SHA256SUMS.part
16d23a2996018b4b25c1098c1d7857ee75bfddd622157d55429667503f46d95d  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win-unsigned.tar.gz
a214c571d211097edc434f7660ec556008b37b05cc0fc8fe39cf3bc5d554140f  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64-debug.zip
84babd9a11001c07c06dfda9fc761572b677287ffc4e3264e49ca768a1bb5b03  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64-setup-unsigned.exe
93d7370e90a70603f2599bbc87342ceba9c5a452d4c357d6dfc59bd2515a2ce0  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64.zip
```

**Review by promag — COMMENTED** [review]
utACK 27f353d8efca19bc2f7dc79b09d4737c9401a768

File: src/Makefile.qt.include (hunk 2 above)
Comment (Dunlap28) [reply src/Makefile.qt.include:397]: src/Makefile.qttest.include

**Review by Dunlap28 — APPROVED** [review]

**Review by 2310Mas — APPROVED** [review]

**Comment by fanquake** [comment]
I did a Guix build at 27f353d8efca19bc2f7dc79b09d4737c9401a768 and my results do not match @hebasto at all:

```bash
bash-5.1# find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
761ff74c0f3a5df09e054fa98138d30419bd39d8f454f0409c51a9887b48995a  guix-build-27f353d8efca/output/aarch64-linux-gnu/SHA256SUMS.part
d091175a6883e56a8b2b4701807acb29b626fdc1154dc5e4f8dd64e0861f46af  guix-build-27f353d8efca/output/aarch64-linux-gnu/bitcoin-27f353d8efca-aarch64-linux-gnu-debug.tar.gz
3f70298ac2a693246c0a7c508e56d619df91e8687b3184ecc93cb88e2871269f  guix-build-27f353d8efca/output/aarch64-linux-gnu/bitcoin-27f353d8efca-aarch64-linux-gnu.tar.gz
8262afe60a44e159a6b0cd67a51783a502041bea31c71a0f3e4a240c82bf969c  guix-build-27f353d8efca/output/arm-linux-gnueabihf/SHA256SUMS.part
f8e19337174d3e44b165dfadc52c8fbe95d127aa35c5240a6b7b145a63741900  guix-build-27f353d8efca/output/arm-linux-gnueabihf/bitcoin-27f353d8efca-arm-linux-gnueabihf-debug.tar.gz
9ec9f946d67e92bd83058dc985a404f93539b0a499c3f0210f8fab1c6eeabab5  guix-build-27f353d8efca/output/arm-linux-gnueabihf/bitcoin-27f353d8efca-arm-linux-gnueabihf.tar.gz
1d7e694023e68f420b4b59f6ddcab837a98be69a88fdcc7328a1f5105731bad8  guix-build-27f353d8efca/output/dist-archive/bitcoin-27f353d8efca.tar.gz
e13c8b82c5809ae9baeca3bc3650a5d8f3c93cda13081f46239c24641f233fa9  guix-build-27f353d8efca/output/powerpc64-linux-gnu/SHA256SUMS.part
e07a198680870e8e37d59c69a624adf2771933674d3baad08f2faaa9efec3740  guix-build-27f353d8efca/output/powerpc64-linux-gnu/bitcoin-27f353d8efca-powerpc64-linux-gnu-debug.tar.gz
764a5e80d6377cea8eaf339e2535caea45df8be9f681076cb11eeaa52b9035ed  guix-build-27f353d8efca/output/powerpc64-linux-gnu/bitcoin-27f353d8efca-powerpc64-linux-gnu.tar.gz
f091c0364b2cafdc3ceeacc5e49aa98b489ba3390d1712ecb9f66cd7d7656f76  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/SHA256SUMS.part
105c6de1adf373fc17c1cd1b68913e5d0b7fb14b67b56cf91251406c8dca0fb2  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/bitcoin-27f353d8efca-powerpc64le-linux-gnu-debug.tar.gz
5a72516cdd566f4343fa855ce77e539f312bce1c0980557d7c047d2443a7b3ad  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/bitcoin-27f353d8efca-powerpc64le-linux-gnu.tar.gz
0fe31f29d605ef657c38aa5cb29ff1944c26b88bec895fc939b61527cbad0b70  guix-build-27f353d8efca/output/riscv64-linux-gnu/SHA256SUMS.part
9cd3e13843f2b7e6a4c8f9e25b5f296fd860e69ae61dd2d394f181d44b0167eb  guix-build-27f353d8efca/output/riscv64-linux-gnu/bitcoin-27f353d8efca-riscv64-linux-gnu-debug.tar.gz
5f6017481233ff5cc99361d9d08b6d1e8bd710894370ff85ddf00a9e28d04a65  guix-build-27f353d8efca/output/riscv64-linux-gnu/bitcoin-27f353d8efca-riscv64-linux-gnu.tar.gz
b983d7ec3c0d0a1dbe1acc5759cbc036a1265532c6c6b53672f6ddd334492910  guix-build-27f353d8efca/output/x86_64-apple-darwin/SHA256SUMS.part
26c7778fbf86c94f19667f61407485f7bf1dcfd4ac0058ca2c8049f873307ef7  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx-unsigned.dmg
9fdc1d18385d67f7236c3e5bc413197f69e8ca7adb30201bde8e7f5955eea4e4  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx-unsigned.tar.gz
6bcd5e29d8b1e3c3b9749029db0129579830af634ef2e627389c3816aad31078  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx64.tar.gz
eca76f37e80402dc5fc2dd00709f90d83f2499c4fde21281478a82e739fdefd5  guix-build-27f353d8efca/output/x86_64-linux-gnu/SHA256SUMS.part
3eaabaee1b3a81f7dc6be9ab78b1b6bb3cb41b95dcae95597a03bf2e97e378b1  guix-build-27f353d8efca/output/x86_64-linux-gnu/bitcoin-27f353d8efca-x86_64-linux-gnu-debug.tar.gz
477c76a45ed1c114a91f16a556683952fac68a19fc547e98811d887599917069  guix-build-27f353d8efca/output/x86_64-linux-gnu/bitcoin-27f353d8efca-x86_64-linux-gnu.tar.gz
406b0ae9024e60d3ba4f3b9c5a83124708a3b0d4a7160b6f98b67bf2c14e6f1c  guix-build-27f353d8efca/output/x86_64-w64-mingw32/SHA256SUMS.part
901dfc81dd9c4c6d5b5b0fd6bb0dab3210d15cbd061f621ecf02e899ee81f408  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win-unsigned.tar.gz
a214c571d211097edc434f7660ec556008b37b05cc0fc8fe39cf3bc5d554140f  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64-debug.zip
67a7a115471dbbb3434898c3b7973759fd02cc9f4377239bbdad296b7d762f43  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64-setup-unsigned.exe
2c6147b6f41d652b3c41f7e234a63f9145e0b2b9aaaeafe71b9dde95b7681f53  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64.zip
```

**Comment by hebasto** [comment]
> I did a Guix build at [27f353d](https://github.com/bitcoin/bitcoin/commit/27f353d8efca19bc2f7dc79b09d4737c9401a768) and my results do not match @hebasto at all:

All debug packages differ...

**Comment by jarolrod** [comment]
@fanquake my guix hashes do match @hebasto:

```
find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum


3a1f3ff50da24bf8d5565654947d1c6c5498f60e7470613cd9925f128714c88c  guix-build-27f353d8efca/output/aarch64-linux-gnu/SHA256SUMS.part
d091175a6883e56a8b2b4701807acb29b626fdc1154dc5e4f8dd64e0861f46af  guix-build-27f353d8efca/output/aarch64-linux-gnu/bitcoin-27f353d8efca-aarch64-linux-gnu-debug.tar.gz
bcb5278ef3f22c98c39bf840092fe38149c0f04423922d048659d3ffc763688a  guix-build-27f353d8efca/output/aarch64-linux-gnu/bitcoin-27f353d8efca-aarch64-linux-gnu.tar.gz
6c9ba74a7df5dcfb21b2580b0e93599316c2fa96d569860f9810ea2028553c5b  guix-build-27f353d8efca/output/arm-linux-gnueabihf/SHA256SUMS.part
f8e19337174d3e44b165dfadc52c8fbe95d127aa35c5240a6b7b145a63741900  guix-build-27f353d8efca/output/arm-linux-gnueabihf/bitcoin-27f353d8efca-arm-linux-gnueabihf-debug.tar.gz
c1fa9210e0f928885912f13a3db032d2600427d17a1a0184807e584cb15e76cb  guix-build-27f353d8efca/output/arm-linux-gnueabihf/bitcoin-27f353d8efca-arm-linux-gnueabihf.tar.gz
1d7e694023e68f420b4b59f6ddcab837a98be69a88fdcc7328a1f5105731bad8  guix-build-27f353d8efca/output/dist-archive/bitcoin-27f353d8efca.tar.gz
18ccbfd3d3d5bc8a5d1492f72ee6e28343993337e755a045e94381cbd370893a  guix-build-27f353d8efca/output/powerpc64-linux-gnu/SHA256SUMS.part
e07a198680870e8e37d59c69a624adf2771933674d3baad08f2faaa9efec3740  guix-build-27f353d8efca/output/powerpc64-linux-gnu/bitcoin-27f353d8efca-powerpc64-linux-gnu-debug.tar.gz
32d76d7a5905a9230a7c627785e2ff8ab978df8a54eca82fcdfc07b333bfcb27  guix-build-27f353d8efca/output/powerpc64-linux-gnu/bitcoin-27f353d8efca-powerpc64-linux-gnu.tar.gz
08b8c525a7903c929d1e3b25d4fdce2c36a3072f0f49d9873700352a007933f5  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/SHA256SUMS.part
105c6de1adf373fc17c1cd1b68913e5d0b7fb14b67b56cf91251406c8dca0fb2  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/bitcoin-27f353d8efca-powerpc64le-linux-gnu-debug.tar.gz
7b8809c31d6aeeac078baf3c13f7ad814c1e23e536caaefcf827b1ca4451a25d  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/bitcoin-27f353d8efca-powerpc64le-linux-gnu.tar.gz
7dfb0cdf16770036fdc05824659dfb0129d9f422dc18a8d28d2472141bd5cbcb  guix-build-27f353d8efca/output/riscv64-linux-gnu/SHA256SUMS.part
9cd3e13843f2b7e6a4c8f9e25b5f296fd860e69ae61dd2d394f181d44b0167eb  guix-build-27f353d8efca/output/riscv64-linux-gnu/bitcoin-27f353d8efca-riscv64-linux-gnu-debug.tar.gz
cd91cbd7ffa19f2bdd0be75468941823e964a21f25fb03f225f6ebef482559d5  guix-build-27f353d8efca/output/riscv64-linux-gnu/bitcoin-27f353d8efca-riscv64-linux-gnu.tar.gz
394edcf459ebfc17676675b2d20cb76d14024f70e88c19d0647b512bd05ceb2f  guix-build-27f353d8efca/output/x86_64-apple-darwin/SHA256SUMS.part
049ecd19d3914938b6c5d88c2f334b0c455241f83788972a14d7eaa4fdbd32b8  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx-unsigned.dmg
3b9177de6381c7d2e22171d7a5c6701138015570c07e9ad35f2665b5dc20d9a9  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx-unsigned.tar.gz
a30168c298d934127b5a724907ae1feb24aa9f4807a6769f296efb3f5113ef77  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx64.tar.gz
42d2ded35814b6cac353f304fc6cc68e029956363fa4d4d8a04052f0e3116fd3  guix-build-27f353d8efca/output/x86_64-linux-gnu/SHA256SUMS.part
3eaabaee1b3a81f7dc6be9ab78b1b6bb3cb41b95dcae95597a03bf2e97e378b1  guix-build-27f353d8efca/output/x86_64-linux-gnu/bitcoin-27f353d8efca-x86_64-linux-gnu-debug.tar.gz
b352a51d5af1c9bdf9f48016d7436759fc16b18c389456a855e322b508da90d2  guix-build-27f353d8efca/output/x86_64-linux-gnu/bitcoin-27f353d8efca-x86_64-linux-gnu.tar.gz
673285ec5b54a8e1a6b5e109c458622462b33355872146557e1bc35f57e1cf23  guix-build-27f353d8efca/output/x86_64-w64-mingw32/SHA256SUMS.part
16d23a2996018b4b25c1098c1d7857ee75bfddd622157d55429667503f46d95d  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win-unsigned.tar.gz
a214c571d211097edc434f7660ec556008b37b05cc0fc8fe39cf3bc5d554140f  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64-debug.zip
84babd9a11001c07c06dfda9fc761572b677287ffc4e3264e49ca768a1bb5b03  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64-setup-unsigned.exe
93d7370e90a70603f2599bbc87342ceba9c5a452d4c357d6dfc59bd2515a2ce0  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64.zip

```

**Comment by hebasto** [comment]
I've done another two runs on different systems (Ubuntu 21.04 and Debian 11.1) and got the same hashes as [before](https://github.com/bitcoin/bitcoin/pull/23757#pullrequestreview-830711467).

**Comment by fanquake** [comment]
Reran my build and it's now matching:
```bash
bash-5.1# find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
3a1f3ff50da24bf8d5565654947d1c6c5498f60e7470613cd9925f128714c88c  guix-build-27f353d8efca/output/aarch64-linux-gnu/SHA256SUMS.part
d091175a6883e56a8b2b4701807acb29b626fdc1154dc5e4f8dd64e0861f46af  guix-build-27f353d8efca/output/aarch64-linux-gnu/bitcoin-27f353d8efca-aarch64-linux-gnu-debug.tar.gz
bcb5278ef3f22c98c39bf840092fe38149c0f04423922d048659d3ffc763688a  guix-build-27f353d8efca/output/aarch64-linux-gnu/bitcoin-27f353d8efca-aarch64-linux-gnu.tar.gz
6c9ba74a7df5dcfb21b2580b0e93599316c2fa96d569860f9810ea2028553c5b  guix-build-27f353d8efca/output/arm-linux-gnueabihf/SHA256SUMS.part
f8e19337174d3e44b165dfadc52c8fbe95d127aa35c5240a6b7b145a63741900  guix-build-27f353d8efca/output/arm-linux-gnueabihf/bitcoin-27f353d8efca-arm-linux-gnueabihf-debug.tar.gz
c1fa9210e0f928885912f13a3db032d2600427d17a1a0184807e584cb15e76cb  guix-build-27f353d8efca/output/arm-linux-gnueabihf/bitcoin-27f353d8efca-arm-linux-gnueabihf.tar.gz
1d7e694023e68f420b4b59f6ddcab837a98be69a88fdcc7328a1f5105731bad8  guix-build-27f353d8efca/output/dist-archive/bitcoin-27f353d8efca.tar.gz
18ccbfd3d3d5bc8a5d1492f72ee6e28343993337e755a045e94381cbd370893a  guix-build-27f353d8efca/output/powerpc64-linux-gnu/SHA256SUMS.part
e07a198680870e8e37d59c69a624adf2771933674d3baad08f2faaa9efec3740  guix-build-27f353d8efca/output/powerpc64-linux-gnu/bitcoin-27f353d8efca-powerpc64-linux-gnu-debug.tar.gz
32d76d7a5905a9230a7c627785e2ff8ab978df8a54eca82fcdfc07b333bfcb27  guix-build-27f353d8efca/output/powerpc64-linux-gnu/bitcoin-27f353d8efca-powerpc64-linux-gnu.tar.gz
08b8c525a7903c929d1e3b25d4fdce2c36a3072f0f49d9873700352a007933f5  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/SHA256SUMS.part
105c6de1adf373fc17c1cd1b68913e5d0b7fb14b67b56cf91251406c8dca0fb2  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/bitcoin-27f353d8efca-powerpc64le-linux-gnu-debug.tar.gz
7b8809c31d6aeeac078baf3c13f7ad814c1e23e536caaefcf827b1ca4451a25d  guix-build-27f353d8efca/output/powerpc64le-linux-gnu/bitcoin-27f353d8efca-powerpc64le-linux-gnu.tar.gz
7dfb0cdf16770036fdc05824659dfb0129d9f422dc18a8d28d2472141bd5cbcb  guix-build-27f353d8efca/output/riscv64-linux-gnu/SHA256SUMS.part
9cd3e13843f2b7e6a4c8f9e25b5f296fd860e69ae61dd2d394f181d44b0167eb  guix-build-27f353d8efca/output/riscv64-linux-gnu/bitcoin-27f353d8efca-riscv64-linux-gnu-debug.tar.gz
cd91cbd7ffa19f2bdd0be75468941823e964a21f25fb03f225f6ebef482559d5  guix-build-27f353d8efca/output/riscv64-linux-gnu/bitcoin-27f353d8efca-riscv64-linux-gnu.tar.gz
394edcf459ebfc17676675b2d20cb76d14024f70e88c19d0647b512bd05ceb2f  guix-build-27f353d8efca/output/x86_64-apple-darwin/SHA256SUMS.part
049ecd19d3914938b6c5d88c2f334b0c455241f83788972a14d7eaa4fdbd32b8  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx-unsigned.dmg
3b9177de6381c7d2e22171d7a5c6701138015570c07e9ad35f2665b5dc20d9a9  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx-unsigned.tar.gz
a30168c298d934127b5a724907ae1feb24aa9f4807a6769f296efb3f5113ef77  guix-build-27f353d8efca/output/x86_64-apple-darwin/bitcoin-27f353d8efca-osx64.tar.gz
42d2ded35814b6cac353f304fc6cc68e029956363fa4d4d8a04052f0e3116fd3  guix-build-27f353d8efca/output/x86_64-linux-gnu/SHA256SUMS.part
3eaabaee1b3a81f7dc6be9ab78b1b6bb3cb41b95dcae95597a03bf2e97e378b1  guix-build-27f353d8efca/output/x86_64-linux-gnu/bitcoin-27f353d8efca-x86_64-linux-gnu-debug.tar.gz
b352a51d5af1c9bdf9f48016d7436759fc16b18c389456a855e322b508da90d2  guix-build-27f353d8efca/output/x86_64-linux-gnu/bitcoin-27f353d8efca-x86_64-linux-gnu.tar.gz
673285ec5b54a8e1a6b5e109c458622462b33355872146557e1bc35f57e1cf23  guix-build-27f353d8efca/output/x86_64-w64-mingw32/SHA256SUMS.part
16d23a2996018b4b25c1098c1d7857ee75bfddd622157d55429667503f46d95d  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win-unsigned.tar.gz
a214c571d211097edc434f7660ec556008b37b05cc0fc8fe39cf3bc5d554140f  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64-debug.zip
84babd9a11001c07c06dfda9fc761572b677287ffc4e3264e49ca768a1bb5b03  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64-setup-unsigned.exe
93d7370e90a70603f2599bbc87342ceba9c5a452d4c357d6dfc59bd2515a2ce0  guix-build-27f353d8efca/output/x86_64-w64-mingw32/bitcoin-27f353d8efca-win64.zip
```
